1-mandatory/4-budgets.js

```diff
@@ -10,7 +10,14 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
-function getBudgets(peopleArray) {}
+function getBudgets(peopleArray) {
+  let NewpeopleArray = peopleArray.map((people) => people.budget);
+  let result =0;
+  for (i = 0; i < NewpeopleArray.length; i++) {
```

Comment:
The logic for the code is perfect. Is there another type of loop that is more suitable for this case?